src/metabase/cmd.clj

```diff
@@ -133,33 +133,17 @@
         (for [[k v] (partition 2 args)]
           [(mbql.u/normalize-token (subs k 2)) v])))
 
-(defn- resolve-enterprise-command [symb]
-  (try
-    (classloader/require (symbol (namespace symb)))
-    (resolve symb)
-    (catch Throwable e
-      (throw (ex-info (trs "The ''{0}'' command is only available in Metabase Enterprise Edition." (name symb))
-                      {:command symb}
-                      e)))))
-
-;;; [[load]] and [[dump]] are the SerDes v1 versions. [[import]] and [[export]] are SerDes v2 versions of [[load]]
-;;; and [[dump]]. The v1 versions are deprecated.
-
-(defn- import*
```

Comment:
The idea here is that when the user runs the v1 dump command, we just call the v1 dump command, and the same for export, v2, etc.

Previously both the `dump` and `export` commands would pass all of their respective arguments to `export*` which would then kind-of-but-not-really parse options for both v1 and v2 dump, then call the EE impl `dump` function with the big map of options, which would in turn decide whether to call v1 or v2.

Reply:
You're finding the scars from the fact that v2's commands were originally also "load" and "dump", but were changed to "import" and "export" at the end of 45, and I didn't want to take on the time or risk of changing this `cmd` infrastructure.